controllers/installation_controller.go

```diff
@@ -120,6 +122,49 @@ var copyArtifactsJob = &batchv1.Job{
 	},
 }
 
+// copyHostPreflightResultsJob is a job we create everytime we need to copy
+// host preflight results from a newly added node in the cluster. Host preflight
+// are run on installation, join or restore operations. The results are stored
+// in /var/lib/embedded-cluster/support/host-preflight-results.json. During a
+// reconcile cycle we will populate the node selector, any env variables and labels.
+var copyHostPreflightResultsJob = &batchv1.Job{
+	ObjectMeta: metav1.ObjectMeta{
+		Namespace: ecNamespace,
+	},
+	Spec: batchv1.JobSpec{
+		Template: corev1.PodTemplateSpec{
+			Spec: corev1.PodSpec{
+				ServiceAccountName: "embedded-cluster-operator",
+				Volumes: []corev1.Volume{
+					{
+						Name: "host",
+						VolumeSource: corev1.VolumeSource{
+							HostPath: &corev1.HostPathVolumeSource{
+								Path: "/var/lib/embedded-cluster",
+								Type: ptr.To[corev1.HostPathType]("Directory"),
+							},
+						},
+					},
+				},
+				RestartPolicy: corev1.RestartPolicyNever,
+				Containers: []corev1.Container{
+					{
+						Name:  "embedded-cluster-updater",
+						Image: "busybox:latest",
```

Comment:
this is not OK
use a static image tag

Reply:
I replace that image [here](https://github.com/replicatedhq/embedded-cluster-operator/blob/6c1bf66212f753d4e3a536688c6d0e82bacfdb89/controllers/installation_controller.go#L1034) which is fed into the pod [here](https://github.com/replicatedhq/embedded-cluster-operator/blob/9e55dc87d92de178bddf54e0d74636cb8a3f4284/charts/embedded-cluster-operator/templates/embedded-cluster-operator-deployment.yaml#L61)